src/CollAction/Helpers/SDImageMetaDataReader.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CollAction.Helpers
+{
+    // System.Drawing.Image implementation of the IImageMetaDataReader interface.
+    public class SDImageMetaDataReader : IImageMetaDataReader
```

Comment:
This feels a bit like premature abstraction. Not everything needs to be wrapped in interfaces and classes, KISS and YAGNI (keep-it-simple and you-aren't-to-need-it).

All abstraction has a price, an overhead wrt reading code, maintenance, refactoring. Since this literally only wraps 3 methods of the Image class, without adding anything, it seems rather superfluous. Also, if you create classes in the future, if you can, make them immutable (http://stackoverflow.com/questions/3769607/why-do-we-need-immutable-class), it's usually easier to program against immutable classes, harder to use them incorrectly.